tests/mir_test_framework/async_server_runner.cpp

```diff
@@ -62,24 +62,35 @@ mtf::AsyncServerRunner::AsyncServerRunner()
     server.add_configuration_option(mtd::logging_opt, mtd::logging_descr, false);
     server.override_the_logger([&]()
         {
-            auto logger = std::make_shared<mtd::MultiLogger>();
+            std::shared_ptr<ml::Logger> result{};
```

Comment:
The postcondition would be clearer with multiple returns of non-null pointers than with multiple conditionals that "have" to initialization this before returning it.

Reply:
Fixed.